cl_sii/rut/__init__.py

```diff
@@ -144,7 +145,10 @@ def __hash__(self) -> int:
     def clean_str(cls, value: str) -> str:
         # note: unfortunately `value.strip('.')` does not remove all the occurrences of '.' in
         #   'value' (only the leading and trailing ones).
-        return value.strip().lstrip('0').replace('.', '').upper()
+        cleaned_value = value.strip().replace('.', '').upper()
```

Comment:
I'm not sure whether *cleaned value* is incorrect or just a valid alternative to *clean value*, but for consistency we may want to use `clean_value`.

Ref: https://github.com/fyntex/lib-cl-sii-python/blob/79b9e85e0276b45260c9b8708658b943c9e09120/cl_sii/rut/__init__.py#L71

Reply:
Changed